### PR TITLE
DI-237 Update the ADR and associated diagrams

### PR DESCRIPTION
**Pull request by ed-sparkes**
## Link to JIRA Ticket

- https://nhsd-jira.digital.nhs.uk/browse/DI-237

## Description

Update the ADR and associated diagrams to reflect that we are no longer using EventBridge Api Destinations

## Type of change

Delete not appropriate

- This change requires a documentation update

### CODE REVIEW

**Review by rohith1122 — APPROVED** [review]
LGTM

**Review by JackPlowman — APPROVED** [review]
LGTM 👍

File: documentation/adr/001-High-level_design_and_tech_stack.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
-* Status: Decided
-* Deciders: The DoS Integration team, the DoS Tech Strategy team, Edward Sparkes
+* Date: **16/01/2022**
+* Status: **Decided**
+* Deciders: **The DoS Integration team, the DoS Tech Strategy team, Edward Sparkes**
 
 ## Context
 
-DI is a greenfiled integration project enabling the changes made by service owners such as Pharmacies to their contact information and opening times oN NHS.uk to be reflected in DoS to ensure consumers of DoS have up to date and accurate information about UEC services.
+DI is a greenfield integration project enabling the changes made by service owners such as Pharmacies to their contact information and opening times oN NHS.uk to be reflected in DoS to ensure consumers of DoS have up to date and accurate information about UEC services.
```
Comment (mabe13) [inline documentation/adr/001-High-level_design_and_tech_stack.md:13]: typo `oN` next to NHS.uk

File: documentation/adr/001-High-level_design_and_tech_stack.md [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
 
 It replaces Profile updater which provided a separate UI for service providers to update data for DoS. The new solution ensures service providers only need to make the changes in 1 place. As an interim solution Profile updater was updated to take a CSV feed from NSH Uk once a day and passed this information to DoS. This solution has its own limitations like data won't reflect to the end user immediately. The proposed solution is based on Messaging Queue technology (publish/subscribe). This paper provides the available options and our recommendations. we are evaluating the options available as "Cloud native solutions/services"  which are  as follows
```
Comment (mabe13) [inline documentation/adr/001-High-level_design_and_tech_stack.md:15]: Another typo not made in this change but worth correcting `NSH Uk`

File: documentation/adr/001-High-level_design_and_tech_stack.md [hunk 3]
```diff
@@ -76,15 +76,15 @@ Cons:
...
+1. Service owners make changes to their service through the NHS Uk website. They can make the changes through a Web user interface or through an API. The API will likely be used by large pharmacy groups such as LLoyds and Boots that manage thousands of pharmacies. Address and organisation name changes cannot be made via the NHS UK website or API and will come via NHS UK from a weekly BSA file uploaded into NHS UK.
 
-2. NHS Uk puts full service record including updated and non updated fields onto two queues. One queue is used internally to update search indexes the other is for DoS integration, there are two separate queues to allow the two consumers to diverge in the messages they need to receive.
+2. NHS Uk updates their record and sends a full service record including updated and non updated fields onto two queues. One queue is used internally to update search indexes the other is for DoS integration, there are two separate queues to allow the two consumers to diverge in the messages they need to receive.
```
Comment (mabe13) [inline documentation/adr/001-High-level_design_and_tech_stack.md:87]: lower case k on `NHS Uk`

**Comment by mabe13** [comment]
Another than a few typos looks good 👍

**Pull request merged**